examples/bridge-app/linux/main.cpp

```diff
@@ -486,6 +546,45 @@ EmberAfStatus HandleReadSwitchAttribute(DeviceSwitch * dev, chip::AttributeId at
     return EMBER_ZCL_STATUS_SUCCESS;
 }
 
+EmberAfStatus HandleReadPowerSourceAttribute(DevicePowerSource * dev, chip::AttributeId attributeId, uint8_t * buffer,
+                                             uint16_t maxReadLength)
+{
+    using namespace app::Clusters;
+    if ((attributeId == PowerSource::Attributes::BatteryChargeLevel::Id) && (maxReadLength == 1))
+    {
+        *buffer = dev->GetBatChargeLevel();
+    }
+    else if ((attributeId == PowerSource::Attributes::Order::Id) && (maxReadLength == 1))
+    {
+        *buffer = dev->GetOrder();
+    }
+    else if ((attributeId == PowerSource::Attributes::Status::Id) && (maxReadLength == 1))
+    {
+        *buffer = dev->GetStatus();
+    }
+    else if ((attributeId == PowerSource::Attributes::Description::Id) && (maxReadLength == 32))
+    {
+        MutableByteSpan zclDescpitionSpan(buffer, maxReadLength);
+        MakeZclCharString(zclDescpitionSpan, dev->GetDescription().c_str());
+    }
+    else if ((attributeId == PowerSource::Attributes::ClusterRevision::Id) && (maxReadLength == 2))
+    {
+        auto rev = ZCL_POWER_SOURCE_CLUSTER_REVISION;
```

Comment:
```suggestion
        uint16_t rev = ZCL_POWER_SOURCE_CLUSTER_REVISION;
```
because we want to be very sure this ends up 2 bytes.